Directory.Packages.props

```diff
@@ -2,14 +2,14 @@
 
   <ItemGroup>
     <PackageVersion Include="AngleSharp" Version="0.13.0" />
-    <PackageVersion Include="JetBrains.Annotations" Version="2022.1.0" />
-    <PackageVersion Include="JustEat.HttpClientInterception" Version="3.1.2" />
+    <PackageVersion Include="JetBrains.Annotations" Version="2023.2.0" />
+    <PackageVersion Include="JustEat.HttpClientInterception" Version="4.0.0" />
     <PackageVersion Include="MartinCostello.Logging.XUnit" Version="0.3.0" />
-    <PackageVersion Include="Microsoft.AspNetCore.Mvc.Testing" Version="8.0.0-rc.1.23421.29" />
-    <PackageVersion Include="Microsoft.AspNetCore.TestHost" Version="8.0.0-rc.1.23421.29" />
-    <PackageVersion Include="Microsoft.IdentityModel.Protocols" Version="6.10.0" />
-    <PackageVersion Include="Shouldly" Version="4.1.0" />
-    <PackageVersion Include="System.Text.Json" Version="8.0.0-rc.1.23419.4" />
+    <PackageVersion Include="Microsoft.AspNetCore.Mvc.Testing" Version="8.0.0-rc.2.23480.2" />
+    <PackageVersion Include="Microsoft.AspNetCore.TestHost" Version="8.0.0-rc.2.23480.2" />
+    <PackageVersion Include="Microsoft.IdentityModel.Protocols" Version="7.0.0" />
+    <PackageVersion Include="Shouldly" Version="4.2.1" />
+    <PackageVersion Include="System.Text.Json" Version="8.0.0-rc.2.23479.6" />
```

Comment:
Not specific at all to this PR, but it's likely we can remove this reference as `System.Text.Json` is inbox since .NET Core 3.0 😃

Reply:
I'll try trimming it out a bit later when I'm sat at a compiler and see what happens 👍